pkg/apis/extensions/types.go

```diff
@@ -87,6 +88,18 @@ type CPUTargetUtilization struct {
 	TargetPercentage int `json:"targetPercentage"`
 }
 
+// Alpha-level support for Custom Metrics in HPA (as annotations).
+type CustomMetricUtilization struct {
```

Comment:
Do we think custom metrics will necessarily be Utilizations?  Do people want to monitor QPS or  (actual QPS over "capacity" QPS)?